ebuild-maintenance/git/text.xml

```diff
@@ -286,11 +275,19 @@ as --force) for the commit.</li>
 </ul>
 
 <p>
-Additionally, some developers prefer referencing bugs on the summary
```

Comment:
We wrote this paragraph when there was a controversy about the summary line vs. the Bug tag. This can go away now.

Reply:
done